frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -348,13 +348,33 @@ export function ActionFilterRow({
                                                 renderValue={(currentValue) => (
                                                     <Tooltip
                                                         title={
-                                                            <>
-                                                                Calculate{' '}
-                                                                {mathDefinitions[math ?? ''].name.toLowerCase()} from
-                                                                property <code>{currentValue}</code>. Note that only{' '}
-                                                                {name} occurences where <code>{currentValue}</code> is
-                                                                set with a numeric value will be taken into account.
-                                                            </>
+                                                            currentValue === '$session_duration' ? (
+                                                                <>
+                                                                    Calculate{' '}
+                                                                    {mathDefinitions[math ?? ''].name.toLowerCase()} of
+                                                                    the Session Duration. Unlike other properties this
```

Comment:
Yeah _but_ it's confusing because to get to that option you go via this which is what confused people...
I think it is worth explicitly mentioning it _or_ someone needs to come up with a general alternative naming.

<img width="289" alt="Screenshot 2023-07-28 at 10 53 42" src="https://github.com/PostHog/posthog/assets/2536520/daeb43cf-9d01-4057-9064-b02d365aa364">

Reply:
Changed it to not mention the properties but I wonder if we need to reconsider the UI here and have session duration as its own thing?

Reply:
Oh yeah, definitely, it should be its own thing